worldedit-core/src/main/java/com/boydti/fawe/object/brush/CatenaryBrush.java

```diff
@@ -34,21 +37,25 @@ public CatenaryBrush(boolean shell, boolean select, boolean direction, double le
     @Override
     public void build(EditSession editSession, BlockVector3 pos2, final Pattern pattern, double size) throws MaxChangedBlocksException {
         boolean visual = editSession.getExtent() instanceof VisualExtent;
+        Player player = editSession.getPlayer();
+        if (player == null) {
+            return; //todo throw error
```

Comment:
We need a proper error message that I add after the fact. It probably should be in a PR when we deal more with brushes. I only added the TODO because I noticed an upstream change that will conflict with this code in the future.